pytorch_to_returnn/naming/naming.py

```diff
@@ -462,10 +464,18 @@ def get_module_by_abs_id_name(self, name: str) -> _types.Module:
       for i, part_name in enumerate(name_parts):
         if part_name[:1] == "(" and part_name[-1:] == ")":
           part_name = part_name[1:-1]
-          possible_namespaces = [name_ for name_ in potential_namespaces if part_name in name_.childs_by_name]
-          assert possible_namespaces, (
+          potential_namespaces_ = []
+          for name_ in potential_namespaces:
+            if part_name in name_.childs_by_name:
+              potential_namespaces_.append(name_.childs_by_name[part_name])
+            else:  # look for explicitly wrapped objects using :class:`Functional`
+              for child_ in name_.childs_by_name.values():
+                for mod_ in child_.modules:
+                  if part_name in getattr(mod_.module, "func_name", ""):
```

Comment:
I think I would not use `getattr` but instead explicitly check for our `Functional` class.